rest-server/src/components/auth/auth.test.js

```diff
@@ -0,0 +1,69 @@
+import request from 'supertest';
+import { dropTables, syncTables } from "../../lib/sqlScripts";
+import app from '../../config/express';
+
+let signupUrl;
+let loginUrl;
+
+beforeAll( async() => {
+  await dropTables();
+  await syncTables();
+  signupUrl = '/api/auth/signup';
+  loginUrl = '/api/auth/login';
+})
+
+describe('POST/api/auth/signup', () => {
+  test('validates email is an actual email', async () => {
+    const response = await request(app)
+      .post(signupUrl)
+      .send({ email: 'notanemail', password: 'password'})
+    expect(response.statusCode).toBe(400);
+  })
+  test('validates password matches custom regex', async () => {
+    const response = await request(app)
+      .post(signupUrl)
+      .send({ email: 'emailthatworks@gmail.com', password: '<script></script>' })
+    expect(response.statusCode).toBe(400);
+  })
+  test('checks if email is present on request', async () => {
+    const response = await request(app)
```

Comment:
const { statusCode } = await ...

Reply:
If we're going to be using one or two specific property from something, then let's just destructure it.